### DISCUSSION TITLE
Cannot seem to clone googlesource.com repos

### DISCUSSION THREAD

**Discussion by theoparis** [GitoxideLabs/gitoxide#638] [Q&A] [answered]
```
gix clone https://android.googlesource.com/platform/packages/modules/adb build/adb
Error: Capabilities could not be parsed

Caused by:
    Got unsupported version '2', expected " 2ls-refsfetch=filter ref-in-want sideband-all packfile-uris wait-for-done shallowserver-option"
```

Not sure what exactly is happening here, but I am not able to use gix to clone a google source repository. Not sure if this an issue with gitoxide or something that google needs to fix themselves 😕

**Comment by Byron** [comment, marked as answer]
Thanks for letting me know! I created [this issue](https://github.com/Byron/gitoxide/issues/639) as `gitoxide` definitely strives to support everything that `git` supports as well.

Especially HTTP servers seem to be quite flexible in what they are allowed to reply and more often than not `gitoxide` will fail to parse responses that it has never seen before. But that shall change soon - please subscribe to the issue for updates.
